app/src/main/java/nl/mpcjanssen/simpletask/task/Priority.kt

```diff
@@ -26,89 +26,38 @@ package nl.mpcjanssen.simpletask.task
 
 import java.util.*
 
-enum class Priority constructor(val code: String, private val listFormat: String, private val detailFormat: String,
-                                        private val fileFormat: String) {
-    NONE("-", "   ", "", ""), A("A", "A", "A", "(A)"), B("B", "B", "B", "(B)"), C("C", "C", "C", "(C)"), D("D", "D", "D", "(D)"), E("E", "E", "E", "(E)"), F("F", "F", "F", "(F)"), G("G", "G", "G", "(G)"), H("H", "H", "H", "(H)"), I("I", "I", "I", "(I)"), J("J", "J", "J", "(J)"), K("K", "K", "K", "(K)"), L("L", "L", "L", "(L)"), M("M", "M", "M", "(M)"), N("N", "N", "N", "(N)"), O("O", "O", "O", "(O)"), P("P", "P", "P", "(P)"), Q("Q", "Q", "Q", "(Q)"), R("R", "R", "R", "(R)"), S("S", "S", "S", "(S)"), T("T", "T", "T", "(T)"), U("U", "U", "U", "(U)"), V("V", "V", "V", "(V)"), W("W", "W", "W", "(W)"), X("X", "X", "X", "(X)"), Y("Y", "Y", "Y", "(Y)"), Z("Z", "Z", "Z", "(Z)");
+enum class Priority {
 
-    fun inListFormat(): String {
-        return listFormat
-    }
+    A, B, C, D, E, F, G, H, I, J, K, L, M, N, O, P, Q, R, S, T, U, V, W, X, Y, Z,
+    NONE {
+        override val code: String = "-"
+        override val fileFormat: String = ""
+    };
```

Comment:
Moved NONE to the end because enums are Comparable in the order they're declared, so now we can just sort them natively in the multicomparator, although I did not do that in this PR.